### DISCUSSION TITLE
Workflow

### DISCUSSION THREAD

**Discussion by ghost** [community/community#33110] [Actions]
![IMG_20220915_090353](https://user-images.githubusercontent.com/100292162/190308088-e6ee0de3-2e08-4aa6-bb3b-c3d3c67270ac.jpg)

I dont why  github disabled  my workflow açtion can any one help me..!?

**Comment by airtower-luna** [comment]
You'll have to [ask GitHub support](https://support.github.com/request) about to answer that, and hopefully get it enabled again. Us other users can't help there.

From what I've seen around here that seems to happen if a workflow trips some kind of abuse detection algorithm, for example by running an RDP server in the Actions runner.

Anyway, good luck!
